src/QuadGK.jl

```diff
@@ -13,7 +13,7 @@ if !isdefined(Base, :QuadGK)
 end
 
 using Compat
-using DataStructures
+using DataStructures, LinearAlgebra
```

Comment:
This will need to be `Compat.LinearAlgebra` for compatibility with Julia 0.6.

Reply:
I also thought it was necessary, but worked on my computer with Julia 0.6.  I thought that `Compat` was exporting the `LinearAlgebra` name.

Reply:
It turned out I have [this package](https://github.com/andreasnoack/LinearAlgebra.jl) installed.